### PR TITLE
Update changes branch to be in sync with the main branch

### PR DESCRIPTION
**Pull request by ann0see**

### CODE REVIEW

**Comment by ann0see** [comment]
@ignotus666 can you please have a look at the conflicts?

File: wiki/es/es-Installation-for-Linux.md [hunk 1]
```diff
@@ -29,22 +29,16 @@ tar -xvf latest.tar.gz
...
-
-```shell
-sudo apt-get install build-essential qtdeclarative5-dev qt5-default qttools5-dev-tools libqt5concurrent5 libjack-jackd2-dev
+sudo apt-get install build-essential qt5-qmake qtdeclarative5-dev qt5-default qttools5-dev-tools libjack-jackd2-dev 
 ```
 
 En **Fedora**:
 
 ```shell
-sudo dnf install qt5-qtdeclarative-devel jack-audio-connection-kit-dbus libqt5concurrent5 jack-audio-connection-kit-devel
```
Comment (ann0see) [inline wiki/es/es-Installation-for-Linux.md:47]: why has this been removed?

**Review by ann0see — COMMENTED** [review]
There are still some things to be discussed.

File: wiki/en/en-Tips-Tricks-More.md [hunk 2]
```diff
@@ -104,12 +104,4 @@ The volume faders in the client's mixer window can be controlled using a MIDI co
 
 So for example, if you're using a Behringer X-Touch, sending MIDI on channel 1 and leaving the offset at default, the command would look like this: `--ctrlmidich 1`. If you have a different controller, e.g. sending MIDI on channel 2 and starting with Control Number 30, the command would be as follows: `--ctrlmidich "2;30"`
 
-MIDI CC messages consist of a Control Number, Controller Value, and Channel. Jamulus listens to the Control Number to know what fader to move, on the channel you specify when launching it with `--ctrlmidich`.
```
Comment (ann0see) [inline wiki/en/en-Tips-Tricks-More.md]: I've now just removed it

File: wiki/es/es-Installation-for-Linux.md (hunk 1 above)
Comment (ignotus666) [reply wiki/es/es-Installation-for-Linux.md:47]: The dependency list? If you mean that, it's in the document - I don't know why it's recorded as having been erased, but it's there (I just checked).

Comment (ann0see) [reply wiki/es/es-Installation-for-Linux.md:47]: Ok. So I will review it and if it's removed will re-add it.

**Pull request merged**